server/pbench/bin/pbench-rsync-satellite

```diff
@@ -133,7 +129,7 @@ for host in $hosts ;do
         # delete all that check out
         for x in $(sed -n '/OK$/s/: OK//p' $logdir/$host/md5-checks.log) ;do
             echo $x $x.md5
-        done | ssh $remotehost "cd $remotearchive/$host; xargs rm" > $logdir/$host/rm.log 2>&1
+        done | ssh $remotehost "cd $remotearchive/$host; python3 /opt/pbench-server/bin/pbench-satellite-cleanup" > $logdir/$host/rm.log 2>&1
```

Comment:
It's not guaranteed that the remote will have python3. It's probably better to just execute it and assume that the installation did the right thing (made it executable and arranged the right shebang line (python2, python3, scl).